frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts

```diff
@@ -0,0 +1,67 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { NooBaaObjectBucketClaimModel } from 'packages/noobaa-storage-plugin/src/models';
+import {
+  storageClassDropDown,
+  defaultSC,
+  scDropdown,
+  createBtn,
+} from '../views/bucketClaimCreation.view';
+import {
+  obcCount,
+  clusterHealth,
+  efficiencyValue,
+  savingsValue,
+} from '../views/noobaaDashboardPage.view';
+
+const BROWSER_TIMEOUT = 30000;
+
+describe('Test Object Service Dashboard Page', () => {
+  it('Create an OBC and test equality', async () => {
+    await browser.get(`${appHost}/dashboards/object-service`);
+    await browser.wait(until.visibilityOf(obcCount));
+    await browser.sleep(5000);
+    let text = await obcCount.getText();
+    const initialCount = text.substring(0, 1);
+    // Go to create form page
+    await browser.get(
+      `${appHost}/k8s/ns/${testName}/${referenceForModel(NooBaaObjectBucketClaimModel)}/~new/form`,
+    );
+    await browser.wait(until.visibilityOf(scDropdown));
+    await storageClassDropDown.click();
+    await browser.wait(until.visibilityOf(defaultSC), BROWSER_TIMEOUT);
+    await defaultSC.click();
+    await createBtn.click();
+
+    await browser.get(`${appHost}/dashboards/object-service`);
+    await browser.wait(until.visibilityOf(obcCount));
+    await browser.sleep(5000);
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+  });
+
+  it('Checks whether MCG is healthy', async () => {
```

Comment:
The order of the test should not matter. If unhealthy even creation of OBC will fail.

Reply:
Yes, it shouldn't matter. But, the test report will look cleaner and can suggest on corner cases.
I just saw a bug where Health Check failed after creating OBC.

In fact, I think we should do a Health Check in beforeAll and afterAll to make sure we started and ended healthy. What do you think?

Reply:
I guess the objective of the test is to catch an unhealthy state at some point. I don't think if an unhealthy MCG creates an OBC that is an issue. If unhealthy MCGs should not be able to create OBCs. We should file a separate bug for that. 
Should we really worry about Health being degraded at a certain point? Again the question that arises is what are we testing? If it's whether health can be affected by other components or is it that all the components are actually working as they are supposed to? Because if we are testing the health parameter extensively then we need to take in all the variables that affect Health. 
If Health is degraded when creating an OBC(or any other resource) then should that be concerning from a UI perspective? Technically the UI is still intact and it's reporting a state of the resource correctly.  The reason why it's health got degraded IMO is out of the scope.